### PR TITLE
return time and date formatted

### PR DESCRIPTION
**Pull request by luciagirasoles**
- function formatdate(someDate) returns: "Today", "Yesterday" or "Monday, May 20th"
- function formatAMPM(date) returns: "12:00am", "12:00pm"

### CODE REVIEW

File: public/assets/js/main.js [hunk 1]
```diff
@@ -40,3 +40,113 @@ function changeActiveChannel(channelName) {
...
+  return days[date.getDay()];
+}
+
+function getMonth(date) {
+  let months = [
+    "Jan",
+    "Feb",
+    "Mar",
+    "Apr",
+    "Jun",
```
Comment (cristianbgp) [inline public/assets/js/main.js]: Be careful, May is a month too :sweat_smile:

Comment (luciagirasoles) [reply public/assets/js/main.js]: made it

File: public/assets/js/main.js [hunk 2]
```diff
@@ -40,3 +40,114 @@ function changeActiveChannel(channelName) {
...
   }
 }
+
+//new Date(year, month, date, hours, minutes, seconds, ms)
+function formatAMPM(date) {
+  var hours = date.getHours();
+  var minutes = date.getMinutes();
+  var ampm = hours >= 12 ? "pm" : "am";
+  hours = hours % 12;
+  hours = hours ? hours : 12; // the hour '0' should be '12'
```
Comment (liannivin) [inline public/assets/js/main.js]: Great work implementing this.

File: public/assets/js/main.js [hunk 3]
```diff
@@ -40,3 +40,114 @@ function changeActiveChannel(channelName) {
     return (activeChannel = channels[channelName]);
   }
 }
+
+//new Date(year, month, date, hours, minutes, seconds, ms)
+function formatAMPM(date) {
+  var hours = date.getHours();
+  var minutes = date.getMinutes();
+  var ampm = hours >= 12 ? "pm" : "am";
```
Comment (yummta) [inline public/assets/js/main.js]: You use "var" and "let" in this file of the same way. In this link you could see the difference between var and let to you would be decide what to use where and when.

File: public/assets/js/main.js [hunk 4]
```diff
@@ -40,3 +40,114 @@ function changeActiveChannel(channelName) {
...
+    "nd",
+    "rd",
+    "th",
+    "th",
+    "th",
+    "th",
+    "th",
+    "th",
+    "th",
+    "st"
```
Comment (yummta) [inline public/assets/js/main.js]: You could declarate ordinals, get WeekDay, getMonth variables like constants out of the function and use the constants whe you needed.

**Pull request merged**